ext/project_data.yaml

```diff
@@ -39,7 +39,8 @@ gem_platform_dependencies:
       win32-process: '~> 0.7.4'
       # Use of win32-security is deprecated
       win32-security: '~> 0.2.5'
-      win32-service: '~> 0.8.6'
+      # win32-service gem is pinned due to PUP-6445
+      win32-service: '= 0.8.7'
```

Comment:
I think this is OK... but I'm not sure about Ruby compat here - have you verified the differences between `0.8.6` and `0.8.7` to make sure there are no implications there?

https://github.com/djberg96/win32-service/compare/win32-service-0.8.6...win32-service-0.8.7